src/community/infix-eng-rev.ts

```diff
@@ -0,0 +1,22 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript } from "../utils";
+import type Decimal from "break_infinity.js";
+
+export class InfixEngineeringReverseNotation extends AbstractInfixNotation {
+  public readonly name = "Reverse infix engineering";
+
+  protected formatMantissa(digit: number): string {
+    return toSubscript(digit);
+  }
+
+  protected formatExponent(exp: number): string {
+    return exp.toString(10);
+  }
+
+  public formatNegativeDecimal(value: Decimal, places: number): string {
```

Comment:
Codacy found an issue: [Declaration of public instance method not allowed after declaration of protected instance method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)